dotcom-rendering/src/server/dev-server.ts

```diff
@@ -39,6 +41,9 @@ export const devServer = (): Handler => {
 			case '/FrontJSON':
 				return handleFrontJson(req, res, next);
 			default: {
+				// Do not redirect assets urls
+				if (req.url.match(ASSETS_URL)) return next();
```

Comment:
A little drive-by bugfix/improvement, trying to load a 404ing assets URL would try and redirect you to a front... which would then error because the json was invalid.